src/Config/Auth.php

```diff
@@ -203,26 +203,36 @@ class Auth extends BaseConfig
      * The validation rules for username
      * --------------------------------------------------------------------
      *
-     * @var string[]
+     * Do not use string rules like `required|valid_email`.
```

Comment:
Ok,
shouldn't the changes be applied to the [documentation](https://datamweb.github.io/shield/customization/validation_rules/)?
```php
//     'rules' => 'required|max_length[30]|min_length[3]|regex_match[/\A[a-zA-Z0-9\.]+\z/]',
'rules' => 'required|max_byte[72]',
// and ...

```

Reply:
Done.